features/google/src/model/importers/access-log-importer.js

```diff
@@ -0,0 +1,45 @@
+import AccessLogEntry from "../entities/access-log-entry";
+
+const accessLogRegex = /\/Access Log Activity\/.*\.csv$/;
```

Comment:
Same as in the other #1526, this name can change. I would at least prepare it for German, Spanish is probably not so common.

Reply:
addressed in 1527  https://github.com/polypoly-eu/polyPod/pull/908 :D